src/index.html

```diff
@@ -49,7 +59,7 @@ <h2 class="headline-2">I’m a software developer</h2>
                 <h3 class="works_headline">My Recent Works</h3>
                 <img src="../media/images/indicator.png" alt="indicator"/>
             </div>
-            <div class="works_cards">
+            <div class="cards works_cards margin_bloc">
```

Comment:
## [OPTIONAL]
- [ ] For HTML best practices and semantic tags, I think it would e nice to use the `<ul>` tag to list all your card. like here 👇 
```
   <ul class="cards-section "> <-- This should use grid to layout its content -->
         <li class="card">....content</li>
         <li class="card">....content</li>
         <li class="card">...content</li>
         <li class="card">....content</li>
      <ul>

Here is your current structure
![image](https://user-images.githubusercontent.com/65068771/149530680-813a56c0-8ece-4eb7-9b46-e9cc3bda9375.png)

Reply:
thanks for reviewing ,actualy i'm using ul tags for all card lists, it would be nice if you specify